.github/workflows/import/converters.js

```diff
@@ -0,0 +1,157 @@
+import { unified } from 'unified';
+import remarkParse from 'remark-parse';
+import remarkGridTable from '@adobe/remark-gridtables';
+import { toHast as mdast2hast, defaultHandlers } from 'mdast-util-to-hast';
+import { raw } from 'hast-util-raw';
+import { mdast2hastGridTablesHandler } from '@adobe/mdast-util-gridtables';
+import { toHtml } from 'hast-util-to-html';
+
+import { JSDOM } from 'jsdom';
+
+function toBlockCSSClassNames(text) {
+  if (!text) return [];
+  const names = [];
+  const idx = text.lastIndexOf('(');
+  if (idx >= 0) {
+    names.push(text.substring(0, idx));
+    names.push(...text.substring(idx + 1).split(','));
+  } else {
+    names.push(text);
+  }
+
+  return names.map((name) => name
+    .toLowerCase()
+    .replace(/[^0-9a-z]+/g, '-')
+    .replace(/^-+/, '')
+    .replace(/-+$/, ''))
+    .filter((name) => !!name);
+}
+
+function convertBlocks(dom) {
+  const tables = dom.window.document.querySelectorAll('body > table');
+
+  tables.forEach((table) => {
+    const rows = [...table.querySelectorAll(':scope > tbody > tr, :scope > thead > tr')];
+    const nameRow = rows.shift();
+    const divs = rows.map((row) => {
+      const cols = row.querySelectorAll(':scope > td, :scope > th');
+      // eslint-disable-next-line no-shadow
+      const divs = [...cols].map((col) => {
+        const { innerHTML } = col;
+        const div = dom.window.document.createElement('div');
+        div.innerHTML = innerHTML;
+        return div;
+      });
+      const div = dom.window.document.createElement('div');
+      div.append(...divs);
+      return div;
+    });
+
+    const div = dom.window.document.createElement('div');
+    div.className = toBlockCSSClassNames(nameRow.textContent).join(' ');
+    div.append(...divs);
+    table.parentElement.replaceChild(div, table);
+  });
+}
+
+function makePictures(dom) {
+  const imgs = dom.window.document.querySelectorAll('img');
+  imgs.forEach((img) => {
+    const clone = img.cloneNode(true);
+    clone.setAttribute('loading', 'lazy');
+    // clone.src = `${clone.src}?optimize=medium`;
```

Comment:
did you want to leave this commented out code in?

Reply:
Yes, that's as it is in nexter too 😁